exposee-app/models/video.js

```diff
@@ -16,7 +16,7 @@ export const Video = sequelize.define('Video', {
     allowNull:false
 
   },
-  userId:{
+  user_Id:{
```

Comment:
nit: this should be user_id, so that it matches the format of all other columns!